### ISSUE TITLE
"Invalid PDF structure" when uploading most local files

### ISSUE THREAD

**Issue by audols** [react-pdf-viewer/react-pdf-viewer#1788]
Not sure if anyone else has experienced the same, but when trying to load most local files through default Basic Usage (having the Viewer component inside the Worker), I am experiencing an "Invalid PDF structure" error that is keeping the file from displaying. This is accompanied by multiple "Warning: getHexString" warnings in console.

<img width="205" alt="image" src="https://github.com/user-attachments/assets/73ae5c9c-4d6a-4962-9768-9c04680711c5">
<img width="605" alt="image" src="https://github.com/user-attachments/assets/9133a8ce-8c93-453d-b888-15a413ec9dd9">

For context the code is set up something like this: (I've tried both `pdf.worker.js` and `pdf.worker.min.js`)

```
<Worker workerUrl="https://unpkg.com/pdfjs-dist@3.4.120/build/pdf.worker.js">
    <Viewer
        fileUrl="kbai_ebook_pg1.pdf"
        defaultScale={0.85}
        plugins={[highlightPluginInstance]}
    />
</Worker>
```

And the PDFs I've tried include:
[kbai_ebook_pg1.pdf](https://github.com/user-attachments/files/16237816/kbai_ebook_pg1.pdf)
[kbai_ebook-part-1-compressed.pdf](https://github.com/user-attachments/files/16237818/kbai_ebook-part-1-compressed.pdf)
[kbai_ebook_pg1_repaired.pdf](https://github.com/user-attachments/files/16237862/kbai_ebook_pg1_repaired.pdf)

I know this isn't an issue with filepath, because referencing this PDF by title at the same location displays it without issue:
[DesignofEverydayThings-Ch1.pdf](https://github.com/user-attachments/files/16237870/DesignofEverydayThings-Ch1.pdf)

Any pointers on how to resolve this?

Using create-react-app and `"pdfjs-dist": "^3.4.120"`.

**Comment by audols** [comment]
Resolved through this comment found on the `react-pdf` repo: 
https://github.com/wojtekmaj/react-pdf/issues/527#issuecomment-592697345

tldr; Something like the following: 

```
import pdf from "../assets/pdfs/myDoc.pdf";

<Worker ...>
    <Viewer fileUrl={pdf}
````
